### ISSUE TITLE
Add convenience function to build cohorts automatically

### ISSUE THREAD

**Issue by mmeierer** [bachmannpatrick/CLVTools#147]
As many first time users are not familiar with the concept of customer cohorts, we should discuss the option to include  convenience functions:
[ ] for building cohorts automatically and 
[ ] batch estimating models for these cohorts.

While the manual definition of cohorts based on expert knowledge seems to be preferable. These convenience functions can help to quickly run an initial model for a firm's entire customer base. 

Potential issues to discuss further:
- What to do with customers that are observed  for a relative short time period? 
- Given that we derive the cohort definition based on a customer's first transaction, do we use the entire transaction history for estmating the model or only the last x time periods? See  https://github.com/bachmannpatrick/CLVTools/issues/146#issuecomment-696277094
- ...

**Comment by AdimDrewnik** [comment]
Good idea. I am new to CLVTools and the concept of cohorts and appropriate data preparation for this case is a problem.
